dist/tools/bossa/Makefile

```diff
@@ -0,0 +1,17 @@
+PKG_NAME     = bossa
+PKG_URL      = https://github.com/shumatech/BOSSA
+PKG_VERSION  = 26154375695f345491bba158d57177aa231d6765
+PKG_LICENSE  = BSD-3-Clause
+PKG_BUILDDIR = $(CURDIR)/bin
+
+.PHONY: all
+
+all: git-download
+	@echo "[INFO] compiling bossac from source now"
+	@env -u CXX COMMON_CXXFLAGS="-std=c++11" $(MAKE) BINDIR=$(PKG_BUILDDIR) -C $(PKG_BUILDDIR) strip-bossac
+	@mv $(PKG_BUILDDIR)/bossac $(CURDIR)/bossac
+
+distclean::
```

Comment:
https://www.gnu.org/software/make/manual/html_node/Double_002dColon.html

tl;dr if there are multiple rules with the same name, they must use double colon.